int21.c

```diff
@@ -138,6 +139,22 @@ void int21(uc_engine *uc)
             //printf("\n>>> 0x%x: interrupt: %x, AH = %02x\n", r_ip, 0x21, r_ah);
             break;
 
+        case 0x00: //terminate process
+            {
+                uc_emu_stop(uc);
+            }
+
+        case 0x02: // character output
+            {
+                uint8_t r_dl;
+
+                uc_reg_read(uc, UC_X86_REG_DL, &r_dl);
+
+                printf("%c\n", r_dl);
```

Comment:
Without the newline the character is printed behind the $ of the shell (on linux). The print function 9h is also affected, but while we don't have a shell like dosbox this is a temporary solution